README.md

```diff
@@ -40,6 +40,26 @@ for everyday use. It is aiming for power users with a modern feature mindset.
 - ✅ VT320 Host-programmable and Indicator status line support.
 - ✅ and much more ...
 
+## Installation
+
+`contour` is packaged and available for installation on multiple distributions.
+ - `Fedora` use official [package](https://packages.fedoraproject.org/pkgs/contour-terminal/contour-terminal/)
+
+```sh
+sudo dnf install contour-terminal
+```
+
+ - `Arch` use AUR [package](https://aur.archlinux.org/packages/contour-git)
```

Comment:
We could probably move the Flathub based installation up into this Installation section then as well. Mind, configuration path for Flathub packages is slightly different (by design of Flatpaks).